test/upgrade/run-upgrade.sh

```diff
@@ -4,7 +4,7 @@
 set -vex
 
 # Make sure the test will work at all
-if [ ! -f "${PWD}/docker-compose.yml" ]; then
+if [ ! -f "${PWD}/CONTRIBUTING.md" ]; then
```

Comment:
why this change?

Reply:
The `docker-compose.yml` file has been deleted in this branch, so I decided to go with a file that is highly unlikely to ever be deleted and can indicate that we are in the root of the repo.